Dapper.Tests/EntityFramework/EFContext.cs

```diff
@@ -6,8 +6,21 @@ namespace Dapper.Tests.EntityFramework
 {
     public class EFContext : DbContext
     {
-        public EFContext(DbConnection connection, bool owned = false) : base(connection, owned) { }
+        public EFContext(DbConnection connection, bool owned = false) : base(connection, owned)
+        {
+
+            Configuration.AutoDetectChangesEnabled = false;
+            Configuration.LazyLoadingEnabled = false;
+            Configuration.ProxyCreationEnabled = false;
+            Configuration.ValidateOnSaveEnabled = false;

```

Comment:
These options are not the defaults. I agree this is a valid test, but it should be an independent test and labeled appropriately.